.github/workflows/workflow.yml

```diff
@@ -75,6 +75,7 @@ jobs:
           command: ${{ matrix.command }}
           cloud-url: file://~
           upsert: true
+          downsert: true
```

Comment:
```suggestion
```

Reply:
Done. I've reverted [the workflow commit](https://github.com/pulumi/actions/commit/487ad9f6f0b157d1933172f6812c07e16eefa5e0).